c3web-server/src/main/scala/com/ifunsoftware/c3web/config/Boot.scala

```diff
@@ -29,52 +29,20 @@
  * POSSIBILITY OF SUCH DAMAGE.
  */
 
-package com.ifunsoftware.c3web
-
-import akka.actor.ActorSystem
-import akka.event.Logging
-import akka.stream.ActorFlowMaterializer
-
-import akka.http.scaladsl.Http
-import akka.http.scaladsl.server.Route
-import akka.http.scaladsl.model.StatusCodes._
-import akka.http.scaladsl.server.Directives._
+package com.ifunsoftware.c3web.config
 
+import akka.io.IO
 import com.typesafe.config.ConfigFactory
+import spray.can.Http
 
 /**
  * Main entry-point of the server application that instantiates all components.
  */
 object Boot extends App {
-  implicit val system = ActorSystem()
-  implicit val executor = system.dispatcher
-  implicit val materializer = ActorFlowMaterializer()
 
   val config = ConfigFactory.load()
-  val logger = Logging(system, getClass)
-
-  val routes = {
-    logRequestResult("akka-http-microservice") {
-      apiRoute ~ staticRoute
-    }
-  }
-
-  /*
-   * This route serves all API requests.
-   */
-  def apiRoute: Route = {
-    path("ping") {
-      complete(OK, "pong")
-    }
-  }
-
-  /*
-   * This route serves all static content (i.e. HTML, CSS, JS).
-   */
-  def staticRoute: Route =
-    path("") {
-      getFromResource("web/index.html")
-    } ~ getFromResourceDirectory("web")
-
-  Http().bindAndHandle(routes, config.getString("http.interface"), config.getInt("http.port"))
+  val services = ActorSystemBean()
```

Comment:
instantiate actor system in here and pass as an argument to `ActorSystemBean` instance